### PR TITLE
Canisters only notify admins if they contain a meaningful amount of dangerous gas when opened

### LINKED ISSUE #27062: "X opened a canister that contains the following:" admin messages appear even for empty canisters
By "empty" I mean "are not releasing any discernible gas into the atmosphere."

Why is this an issue: Leads to bwoinks that are not necessary, clogs the logs

How to reproduce: I reproduced it by var-editing the gas_mixture contents of a freon canister to 1 (effectively 0, shows up as 0 on the canister) then opening it.

### PR DESCRIPTION
**Pull request by duncathan**
:cl:
tweak: opening a dangerous canister will only alert admins if it contains meaningful amounts of a dangerous gas. No more being bwoinked because you opened an empty canister that used to have plasma in it.
/:cl:
@bgobandit <3

Fixes #27062

### CODE REVIEW

**Comment by Iamgoofball** [comment]
TY

**Comment by duncathan** [comment]
hmm should I have it include the amount in the message too?

**Comment by bgobandit** [comment]
@duncathan it's already on another line so why not

**Comment by duncathan** [comment]
I decided to add it to both so that admins can make more informed decisions about whether it really was significant.

**Comment by bgobandit** [comment]
ok, just saying, some admins (cough LEO Z) hate "unnecessary" info in admin messages

**Comment by duncathan** [comment]
The difference between 20 moles of plasma and 200 moles of plasma is significant enough to warrant this imo

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -395,25 +395,21 @@
...
-						if(n2o)
-							log_admin("N2O")
-							message_admins("N2O")
-						if(bz)
-							log_admin("BZ Gas")
-							message_admins("BZ Gas")
-						if(freon)
-							log_admin("Freon")
-							message_admins("Freon")
+						for(var/name in danger)
```
Comment (coiax) [inline code/modules/atmospherics/machinery/portable/canister.dm]: Please can you just use `english_list` rather than having multiline admin message

**Comment by Alek2ander** [comment]
A canister can be full of benign gases and still cause trouble because of unsafe temperature or pressure

**Comment by Earthykiller127** [comment]
@Alek2ander SSSSHHHHHHHHH

**Comment by Iamgoofball** [comment]
RIP @Alek2ander about to get stickybanned

**Pull request merged**